smarts/core/scenario.py

```diff
@@ -662,7 +662,10 @@ def plane_filepath(self):
 
     @property
     def vehicle_filepath(self):
-        return os.path.join(self._root, "vehicle.urdf")
+        for fname in os.listdir(self._root):
```

Comment:
We are breaking some conventions. Is it possible to return a dict like `{"bus": "bus.urdf", "car": "vehicle.urdf"}`, then in `smarts/core/vehicle.py` no need to do the vehicle_filepath discovery.

Reply:
Yes, now I see why.